anitya/lib/ecosystems/crates.py

```diff
@@ -28,3 +28,4 @@ class CratesEcosystem(BaseEcosystem):
 
     name = 'crates.io'
     default_backend = 'crates.io'
+    librariesio_name = 'Cargo'
```

Comment:
Having to make all the ecosystem plugins libraries.io aware irks me a bit, but I guess the alternative is making the libraries.io handling code aware of all the ecosystems, and that runs somewhat counter to the design of the plugin model.

However, normalising with `str.lower()` would be sufficient to map all of the libraries.io names except `cargo`, and for that we could potentially change the ecosystem name on the Anitya plugin side.

Alternatively (and to better support deciding we want to change canonical ecosystem names in the future), this could be handled by way of a general purpose "aliases" field on the plugins, rather than a libraries.io specific field.